cpp/unreal_plugins/UrdfBot/Source/UrdfBot/UrdfRobotComponent.h

```diff
@@ -20,15 +21,16 @@ struct UrdfJointDesc;
 struct UrdfLinkDesc;
```

Comment:
I realize I've gone back and forth on this, but we can remove the whitespace between `class UUrdfJointComponent;` and `struct UrdfJointDesc;`. I've grouped forward declarations like this throughout the code now and I think it looks better. (Unreal types are still separated from our types with whitespace.)

Reply:
fixed